CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 2.0.1
+
+- `LoggingBugfenderListener.listen()` now returns a
+  `StreamSubscription<LogRecord>` instead of `void`
+- Update README.
+
```

Comment:
Since the API changed I think it deserves a minor version bump (`2.1.0`)

Reply:
Good point. Thanks.